UnityProject/Assets/Scripts/Construction/ConveyorBeltSwitch.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ConveyorBeltSwitch : MonoBehaviour, ICheckedInteractable<HandApply>
+{
+	public SpriteRenderer spriteRenderer;
+
+	public ConveyorBelt[] conveyorBelts;
+
+	public Sprite Forward;
+	public Sprite Backward;
+	public Sprite Off;
+
+	private Sprite LastSprite;
+
+	public int CurrentState = 1; // 0 Backwards, 1 Off, 2 Forwards
+
```

Comment:
Make it a `enum`, don't use `int `for states.

Reply:
You can use it for Conveyor and Switch
![ConveyorStatus](https://user-images.githubusercontent.com/42844726/79425729-ce54b280-7fca-11ea-9896-6d8a6fde3e7f.PNG)